squirrel.frontier/src/test/java/org/dice_research/squirrel/frontier/impl/FrontierImplTest.java

```diff
@@ -126,50 +125,76 @@ public void simlpeRecrawling() throws Exception {
         // Add the URIs to the frontier
         List<CrawleableUri> uris = new ArrayList<>();
         CrawleableUri uri_1 = cuf.create(new URI("http://dbpedia.org/resource/uriThatShouldBeRecrawled"),
-                InetAddress.getByName("127.0.0.1"), UriType.DEREFERENCEABLE);
+            InetAddress.getByName("127.0.0.1"), UriType.DEREFERENCEABLE);
         CrawleableUri uri_2 = cuf.create(new URI("http://dbpedia.org/resource/normalUri"),
-                InetAddress.getByName("127.0.0.1"), UriType.DEREFERENCEABLE);
+            InetAddress.getByName("127.0.0.1"), UriType.DEREFERENCEABLE);
         uris.add(uri_1);
         uris.add(uri_2);
-
         frontier.addNewUris(uris);
-
         List<CrawleableUri> nextUris = frontier.getNextUris();
         for (CrawleableUri uri : nextUris) {
             Assert.assertTrue(uris.contains(uri));
         }
         for (CrawleableUri uri : uris) {
             Assert.assertTrue(nextUris.contains(uri));
         }
-
         // Set the first URI as recrawlable
         for (CrawleableUri uri : nextUris) {
-            if(uri.getUri().equals(uri_1.getUri())) {
+            if (uri.getUri().equals(uri_1.getUri())) {
                 uri.addData(Constants.URI_PREFERRED_RECRAWL_ON, System.currentTimeMillis() - 1);
             }
         }
-
         frontier.crawlingDone(uris);
-        
         uris.add(uri_1);
         uris.add(uri_2);
-
         nextUris = frontier.getNextUris();
         Assert.assertNotNull(nextUris);
         assertTrue("uri_1 has been expected but couldn't be found", nextUris.contains(uri_1));
         Assert.assertEquals(1, nextUris.size());
         assertFalse("uri_2 has been found but was not expected", nextUris.contains(uri_2));
     }
 
+    @Test
+    public void RecrawlingTest() throws Exception {
```

Comment:
This test is not working anymore as expected and needs to be adapted by writing metadata of the "simulated" crawling into the "mimicked" SPARQL endpoint. Otherwise the Frontier will not receive the `uri_1` as recrawlable.